### PR TITLE
add GetArchiveEvents

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: .markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (DarkPark) [inline .markdownlint.json:1]: unintentionally?

File: docs/commands/GetArchiveEvents.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# GetArchiveEvents
```
Comment (DarkPark) [inline docs/commands/GetArchiveEvents.md:1]: would be nice to add new commands to the command list

File: docs/commands/GetArchiveEvents.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+## Request
+
+### Format
+
+| Size   | Type        | Field                                  |
+| ------ | ----------- | -------------------------------------- |
+| `1`    | `byte`      | command id = `0x0b`                    |
+| `1`    | `byte`      | command size = `5`                     |
+| `4`    | `uint32_be` | [start time](../types.md#time-2000) |
```
Comment (DarkPark) [inline docs/commands/GetArchiveEvents.md:14]:
```suggestion
| `4`    | `uint32_be` | [start time](#start-time) |
```

File: docs/commands/GetArchiveEvents.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+| `1`    | `byte`      | [events](#events)                      |
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **start time**
+
+Start time in seconds to read from archive. In [time 2000](../types.md#time-2000) format.
```
Comment (DarkPark) [inline docs/commands/GetArchiveEvents.md:23]:
```suggestion
Start time in seconds to read from archive (in [time 2000](../types.md#time-2000) format).
```
?

File: docs/commands/GetArchiveEvents.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+#### **events**
+
+Amount of events to get from archive.
+
+### Examples
+
+#### channels `1`, `3`, `4`:
```
Comment (DarkPark) [inline docs/commands/GetArchiveEvents.md:31]: seems wrong title

File: docs/commands/GetArchiveEvents.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+| Field        | Value                     | Bits                                 | Hex          |
+| ------------ | ------------------------- | ------------------------------------ | -----------  |
+| command id   | `11`                      |                                      | `0x0b`       |
+| command size | `5`                       |                                      | `0x05`       |
+| start time   | `2023.04.03 14:01:17 GMT` | `0b00101011101111011001100010101101` | `0x2bbd98ad` |
```
Comment (DarkPark) [inline docs/commands/GetArchiveEvents.md:37]: do we need bits here? let's do it as in Time2000.md

maybe we can put both `2023.04.03 14:01:17 GMT` and seconds in Value column

File: docs/commands/GetArchiveEvents.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+| ...    | ...         | ...                                           |
+| `4`    | `uint32_be` | [event `N` time](../types.md#time-2000)       |
+| `1`    | `byte`      | [event `N` id](#event-id)                     |
+| `1`    | `byte`      | [event `N` sequence number](#sequence-number) |
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **event `N` time**
```
Comment (DarkPark) [inline docs/commands/GetArchiveEvents.md:66]:
```suggestion
#### **event `N` time**
```

`N` was not used before in similar cases

File: docs/commands/GetArchiveHoursMC.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+<br>
+[See details](../types.md#packed-hours).
+
+#### **channels bit set**
+
+[See details](../types.md#channels-bit-set).
+
+### Examples
+
+#### channels `1`, `3`, `4`:
```
Comment (DarkPark) [inline docs/commands/GetArchiveHoursMC.md:40]: wrong title

File: .markdownlint.json (hunk 1 above)
Comment (DeadByElpy) [reply .markdownlint.json:1]: deleted

File: docs/commands/GetArchiveEvents.md (hunk 2 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveEvents.md:1]: added

File: docs/commands/GetArchiveEvents.md (hunk 4 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveEvents.md:23]: fixed

File: docs/commands/GetArchiveEvents.md (hunk 3 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveEvents.md:14]: fixed

File: docs/commands/GetArchiveEvents.md (hunk 5 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveEvents.md:31]: fixed

File: docs/commands/GetArchiveEvents.md (hunk 6 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveEvents.md:37]: bits column - deleted, seconds with date representation added

File: docs/commands/GetArchiveHoursMC.md (hunk 8 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveHoursMC.md:40]: fixed

**Review by DarkPark — DISMISSED** [review]

**Review by DarkPark — APPROVED** [review]

**Pull request merged**